services-js/commissions-app/src/server/services/Email.ts

```diff
@@ -0,0 +1,109 @@
+import { Client as PostmarkClient } from 'postmark';
+import Rollbar from 'rollbar';
+
+import EmailContent, {
+  TemplateData as ApplicationData,
+} from '../email/EmailContent';
+
+// see services-js/registry-certs/server/services/Emails.ts
+
+/**
+ * Sends a confirmation email to the applicant, as well as to the policy office.
+ */
+export default class Email {
+  from: string;
+  postmarkClient: PostmarkClient;
+  rollbar: Rollbar;
+  emailContent: EmailContent;
+
+  applicantSubject = 'We’ve received your application';
+  policyOfficeSubject = 'New City Clerk Boards and Commissions Application';
+  policyOfficeToAddress = process.env.POLICY_OFFICE_TO_ADDRESS ||
+  'boardsandcommissions@boston.gov';
+
+  constructor(from: string, postmarkClient: PostmarkClient, rollbar: Rollbar) {
+    this.from = from;
+    this.postmarkClient = postmarkClient;
+    this.rollbar = rollbar;
+
+    this.emailContent = new EmailContent();
+  }
+
+  // todo: need newId from db server
+  public sendConfirmations(validForm, fetchedBoards, applicationId) {
+    const applicationData: ApplicationData = {
+      commissionNames: namesFromIds(fetchedBoards, validForm.commissionIds),
+      firstName: validForm.firstName,
+      lastName: validForm.lastName,
+      email: validForm.email,
+      applicationId,
+    };
+
+    const renderedApplicantBodies = this.emailContent.renderApplicantBodies(
+      applicationData
+    );
+    const renderedPolicyOfficeBodies = this.emailContent.renderPolicyOfficeBodies(
+      applicationData
+    );
+
+    // send to applicant...
+    this.sendConfirmationEmail(
+      applicationData.email,
+      this.applicantSubject,
+      renderedApplicantBodies
+    ).catch(err => this.rollbar.error(`Error sending applicant email: ${err}`));
```

Comment:
I’d recommend passing `err` directly to `this.rollbar.error` so that the stack trace and stuff all get sent.

(Here and below)